docs/pages/eas-update/continuous-deployment.mdx

```diff
@@ -1,35 +1,47 @@
 ---
-title: Continuous Deployment
+title: Continuous deployment
 description: Learn how to use the fingerprint runtime version for continuous deployment in production and during development.
 ---
 
-> **info** This is available in beta for SDK 51 and not yet considered stable. Use may result in unexpected system behavior.
+import { Collapsible } from '~/ui/components/Collapsible';
 
-## Concepts
+## Prerequisites
 
-### Fingerprint runtime versioning
+- You need to have a GitHub account and use it to host your git project.
+- You need to have an Expo token configured in your GitHub repository.
 
-The [`fingerprint` runtime version policy](/eas-update/runtime-versions/#fingerprint-runtime-version-policy) uses the `@expo/fingerprint` package to generate a hash of your project when making a build or publishing an update, and then uses that hash as the runtime version. The hash is calculated based on dependencies, custom native code, native project files, and configuration, amongst other things.
+<Collapsible summary="How to configure the EXPO_TOKEN environment variable on your GitHub repository">
 
-By automatically calculating the runtime version, you don't have to be concerned about native layer compatibility with the JavaScript application when deploying updates to builds.
+- Navigate to [https://expo.dev/settings/access-tokens](https://expo.dev/settings/access-tokens).
+- Click **Create token** to create a new personal access token.
+- Copy the token generated.
+- Navigate to https://github.com/your-username/your-repo-name/settings/secrets/actions by replacing "your-username" and "your-repo-name" with your project's info.
+- Under **Repository secrets**, click **New repository secret**.
+- Create a secret with the name **EXPO_TOKEN**, and paste the copied access token in as the value.
 
-## Continuous Integration (CI) and Continuous Deployment (CD)
+</Collapsible>
```

Comment:
This should probably just be in the action readme? https://github.com/expo/expo-github-action And then we should link there from here?

Reply:
probably! i just moved this around. let's do that in a follow up